rust/feature-flags/src/router.rs

```diff
@@ -22,3 +22,39 @@ where
         .route("/flags", post(v0_endpoint::flags).get(v0_endpoint::flags))
         .with_state(state)
 }
+
+// TODO, eventually we can differentiate read and write postgres clients, if needed
+// I _think_ everything is read-only, but I'm not 100% sure yet
```

Comment:
there shall cometh a write client too 🫠

see the part below this line: https://github.com/PostHog/posthog/blob/master/posthog/models/feature_flag/flag_matching.py#L797